.github/workflows/ci-daily.yml

```diff
@@ -35,12 +35,14 @@ jobs:
           sudo apt-get install libffi7
       - name: Install requirements
         run: |
-          pip install cirq~=1.0.dev &&
-          pip install \
-            -r dev_tools/requirements/deps/format.txt \
-            -r dev_tools/requirements/deps/pylint.txt \
-            -r dev_tools/requirements/deps/pytest.txt \
-            -r dev_tools/requirements/deps/notebook.txt
+          pip install --upgrade cirq~=1.0.dev &&
+            version=$(pip show cirq | sed -n "s/^Version: //p" | grep .) &&
```

Comment:
I wasn't sure what the "grep ." is doing, so I tried using awk – this might be slightly more direct:
 
```bash
$(pip show pygments | awk '/^Version:/ {print $2}')
```

Reply:
Ack.  `grep .` is there to give failing exit status for empty output (ie, pip or sed somehow failed) and to stop execution right there (the `&&` command separator).  Otherwise grep just repeats its input because `.` matches any character.